Automattic/jetpack
# Admin Page: display more info and support link in error message

Fixes #20006

Let's provide more potential debugging steps (refreshing the page) and an option to contact support when folks run into issues in the Jetpack dashboard.Fixes #20006

#### Changes proposed in this Pull Request:

Let's provide more potential debugging steps (refreshing the page) and an option to contact support when folks run into issues in the Jetpack dashboard.

**Before**

![image](https://user-images.githubusercontent.com/390760/120473115-97905600-c39e-11eb-84dd-3d4ccd062bd3.png)

**After**

![image](https://user-images.githubusercontent.com/426388/121184940-dd995e00-c865-11eb-90a7-7a13b0dc55d6.png)


#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

1. Find the `Jetpack_Core_Json_Api_Endpoints::get_site_data()` API endpoint method
2. Add some output into the first line of the method (e.g. `echo 'something';`) to break the output JSON.
3. Load the Jetpack dashboard, and check the notice. Try clicking the link.
